app/src/ui/list.tsx

```diff
@@ -560,7 +594,9 @@ export class List extends React.Component<IListProps, IListState> {
 
     return (
       <Grid
+        aria-label={''}
```

Comment:
Yes, they are. Unfortunately we can't set them to `undefined` since the Grid has `defaultProps` that then take precedence. We can't set to null either because the type definitions don't allow it so this was the best I could come up with. Thinking about this some more though I think perhaps the best solution would be to use `null!` since we really don't want the attributes to render. I'll give that a shot